src/main/resources/db/migration/R__functions.sql

```diff
@@ -55,11 +55,6 @@ BEGIN
 
     SELECT sha256(bytes_) INTO hash_;
 
-    IF EXISTS (SELECT * FROM live_objects WHERE hash = hash_) THEN
```

Comment:
How do we go about this one?
https://datatracker.ietf.org/doc/html/rfc8181#section-2.5

Reply:
We already checked the url being present a few lines before this one.

Reply:
Oh yeah, indeed.